contract/package.json

```
@@ -14,6 +14,7 @@
     "build": "yarn build:deployer",
     "build:deployer": "rollup -c rollup.config.mjs",
     "test": "ava --verbose",
+    "format": "yarn prettier --write .github golang packages",
```

@dckc I tried blindly cargo culting this but it did not work. How to I `yarn format` in this repo?

---

I guess you don't? I don't know.

I use [configure-vscode.sh](https://github.com/Agoric/agoric-sdk/blob/master/scripts/configure-vscode.sh) and format-on-save.

---

in agoric-sdk  the `"format"` is at the repo root level. So step one is to move this to https://github.com/Agoric/dapp-agoric-basics/blob/main/package.json

Step is to update the list of paths to format. E.g.
```suggestion
    "format": "yarn prettier --write .github api contract ui",
```

---

Worked great, thanks!

Done.